lib/howl/command.moon

```diff
@@ -204,8 +204,8 @@ interact.register
       .items = get_command_items!
       .headers = { 'Command', 'Key binding', 'Description' }
       .columns = {
-        { style: 'string' }
-        { style: 'keyword' }
+        { style: 'string', min_width: 30 }
```

Comment:
Perhaps we could detect the actual widths to use instead of using a hard coded value? As it is this almost makes the list not move but not quite.